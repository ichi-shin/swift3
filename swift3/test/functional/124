#!/bin/bash
# Copyright(c)2014 NTT corp. All Rights Reserved.
# DB1-1-4-9_5

. ./common

S3USER=tester

_s3_put /bucket
_s3_put /bucket/source -T /dev/null
_s3_put /bucket2/dest -D - -H 'x-amz-copy-source: /bucket/source' -D - | _filter_curl xml
_s3_get /bucket2/dest -D - | _filter_curl xml

