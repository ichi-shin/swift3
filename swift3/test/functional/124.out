QA output created by 124
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X PUT -T /dev/null http://SWIFT_HOST/bucket/source... 200
> s3curl --id tester -- -X PUT -D - -H x-amz-copy-source: /bucket/source -D - http://SWIFT_HOST/bucket2/dest... 404
HTTP/1.1 404 Not Found
Content-Type: text/xml
Date: DATE
Transfer-Encoding: chunked
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<Error>
  <Code>NoSuchBucket</Code>
  <Message>The specified bucket does not exist.</Message>
  <RequestId>TXID</RequestId>
  <BucketName>bucket2</BucketName>
</Error>
> s3curl --id tester -- -X GET -D - http://SWIFT_HOST/bucket2/dest... 404
HTTP/1.1 404 Not Found
Content-Type: text/xml
Date: DATE
Transfer-Encoding: chunked
x-amz-id-2: TXID
x-amz-request-id: TXID
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<Error>
  <Code>NoSuchBucket</Code>
  <Message>The specified bucket does not exist.</Message>
  <RequestId>TXID</RequestId>
  <BucketName>bucket2</BucketName>
</Error>
